DigitalOffice/Tests/UserServiceTests/ValidatorsTests/UserCreateRequestValidatorTests.cs

```diff
@@ -0,0 +1,200 @@
+﻿using FluentValidation;
+using NUnit.Framework;
+using UserService.RestRequests;
+using UserService.Validators;
+
+namespace UserServiceUnitTests.ValidatorsTests
+{
+    public class UserCreateRequestValidatorTests
+    {
+        private IValidator<UserCreateRequest> validator = new UserCreateRequestValidator();
+
+        [Test]
+        public void ShouldThrowValidationExceptionWhenFirstNameIsEmpty()
+        {
+            var request = new UserCreateRequest
+            {
+                FirstName = "",
+                LastName = "Example",
+                MiddleName = "Example",
+                Email = "Example@gmail.com",
+                Status = "Example",
+                Password = "Example"
+            };
+            Assert.Throws<ValidationException>(() => validator.ValidateAndThrow(request), "First name is empty.");
```

Comment:
Read do-backend-wiki and refactor all tests

Reply:
fixed